jmx-http/src/main/java/io/airlift/jmx/JmxHttpModule.java

```diff
@@ -148,6 +149,7 @@ public void serialize(CompositeData data, JsonGenerator jsonGenerator, Serialize
         }
 
         @Override
+        @SuppressWarnings("deprecation")
```

Comment:
why added now?

Reply:
Jackson was updated with the airbase bump and these methoda are now deprecated

Reply:
why deprecated? should we stop using them?